File: problem-solving/algorithms/dotnet/HackerrankPsAlgorithms0003.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;

class Result
{
    // A Very Big Sum
    // https://www.hackerrank.com/challenges/a-very-big-sum/problem?isFullScreen=true

    public static long aVeryBigSum(List<long> ar)
    {
        // --- You need to fill this method ---
        return ar.Sum();
    }

}

class Solution
{
    public static void Main(string[] args)
    {
        TextWriter textWriter = new StreamWriter(@System.Environment.GetEnvironmentVariable("OUTPUT_PATH"), true);

        int arCount = Convert.ToInt32(Console.ReadLine().Trim());

        List<long> ar = Console.ReadLine().TrimEnd().Split(' ').ToList().Select(arTemp => Convert.ToInt64(arTemp)).ToList();

        long result = Result.aVeryBigSum(ar);

        textWriter.WriteLine(result);

        textWriter.Flush();
        textWriter.Close();
    }
}

File: problem-solving/algorithms/java/HackerrankPsAlgorithms0003.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.FileWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.List;
import java.util.stream.Collectors;
import java.util.stream.Stream;

public class HackerrankPsAlgorithms0003 {
	// A Very Big Sum
	// https://www.hackerrank.com/challenges/a-very-big-sum/problem?isFullScreen=true

	// --- You need to fill this method ---
	public static long aVeryBigSum(List<Long> ar) {
		return ar.stream().mapToLong(Long::longValue).sum();
	}

	public static void main(String[] args) throws IOException {
		BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(System.getenv("OUTPUT_PATH")));

		int arCount = Integer.parseInt(bufferedReader.readLine().trim());

		List<Long> ar = Stream.of(bufferedReader.readLine().replaceAll("\\s+$", "").split(" "))
				.map(Long::parseLong)
				.collect(Collectors.toList());

		long result = aVeryBigSum(ar);

		bufferedWriter.write(String.valueOf(result));
		bufferedWriter.newLine();

		bufferedReader.close();
		bufferedWriter.close();
	}

}
